### PR TITLE
Fixed breadcrumbs params detection

### PR DESCRIPTION
**Pull request by abelgibayev**
- [ ] **Breaking change?** (if so, please describe the impact and migration path for existing applications:)

**What changes did you make?** (Give an overview)

**Is there anything you'd like reviewers to focus on?**

**How Has This Been Tested?** (put an "X" next to an item)

- [ ] No need to
- [x] Manually(please, describe, when necessary)
- [x] Unit checks
- [ ] Integration checks
- [ ] Covered by existing automation

**Checklist** (put an "X" next to an item, otherwise PR will fail)
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation (e.g. **ENVIRONMENT VARIABLES**)
- [x] My changes generate no new warnings(e.g. Sonar is happy)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged

Check out [Contributing](https://github.com/provectus/kafka-ui/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/provectus/kafka-ui/blob/master/CODE-OF-CONDUCT.md)

### CODE REVIEW

**Comment by NelyDavtyan** [comment]
Suggest cover the task with tests, sonar 91.5%.

File: kafka-ui-react-app/src/components/common/Breadcrumb/Breadcrumb.route.tsx [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+export const BreadcrumbRoute: React.FC<RouteProps> = ({
+  children,
+  render,
+  component,
+  ...props
+}) => {
+  return (
+    <Route
+      {...props}
+      render={(routeParams) => {
```
[the remark this answers is not in the document]
Comment (workshur) [reply kafka-ui-react-app/src/components/common/Breadcrumb/Breadcrumb.route.tsx:32]: Line 44

**Review by workshur — APPROVED** [review]
LGTM

**Review by Motherships — APPROVED** [review]

File: kafka-ui-react-app/src/components/common/Breadcrumb/Breadcrumb.route.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (NelyDavtyan) [reply kafka-ui-react-app/src/components/common/Breadcrumb/Breadcrumb.route.tsx:32]: I mean property is not passed from anywhere.
Like this `<BreadcrumbRoute render={something} />`

**Review by NelyDavtyan — APPROVED** [review]

**Pull request merged**